feed/index.html

```diff
@@ -0,0 +1,200 @@
+          <h2>Your feed</h2>
+        </div>
+      </section>
+      <!-- Sort end -->
+
+      <!-- Feed posts  -->
+      <section class="container mt-5 mb-3"></section>
+      <!-- Feed posts end -->
+
+      <!-- Form to submit a new post -->
+      <section
+        class="container col-lg-10 col-xl-8 col-xxl-6 d-flex flex-column"
+      >
+        <h2 class="mt-5 mb-4">Submit your own post</h2>
+        <form id="postForm">
+          <div class="mb-3">
+            <label for="postTitle" class="form-label">Title</label>
+            <input
+              type="text"
+              class="form-control"
+              id="postTitle"
+              placeholder="Your post title"
+              required
+            />
+          </div>
+          <div class="mb-3">
+            <label for="postBody" class="form-label">Post Body</label>
+            <textarea
+              class="form-control"
+              id="postBody"
+              rows="3"
+              placeholder="Your post text here"
+              required
+            ></textarea>
+          </div>
+          <div class="mb-3">
+            <label for="postTags" class="form-label"
+              >Tags (comma-separated)</label
+            >
+            <input
+              type="text"
+              class="form-control"
+              id="postTags"
+              placeholder="tag1, tag2, tag3"
+            />
+          </div>
+          <div class="mb-3">
+            <label for="postMedia" class="form-label"
+              >Media URL (optional)</label
+            >
+            <input
+              type="text"
+              class="form-control"
+              id="postMedia"
+              placeholder="https://url.com/image.jpg"
+            />
+          </div>
+          <div class="m-2 d-flex justify-content-center">
+            <button type="submit" class="btn btn-primary">Submit</button>
+          </div>
+        </form>
+      </section>
+      <!-- form end -->
+
+      <!-- Footer  -->
+      <footer class="py-3 my-4 mt-5">
+        <ul class="nav justify-content-center border-bottom pb-3 mb-3">
+          <!-- <li class="nav-item">
+            <a href="#" class="nav-link px-2">Home</a>
+          </li> -->
+          <li class="nav-item">
+            <a href="../profile/index.html" class="nav-link px-2">Profile</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link px-2">Feed</a>
+          </li>
+        </ul>
+        <p class="text-center">© 2023 CodeSharing, Inc</p>
+      </footer>
+      <!-- Footer end -->
+    </main>
+    <!-- Enables sorting text  -->
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ygbV9kiqUc6oa4msXn9868pTtWMgiQaeYH7/t7LECLbyPA2x65Kgf80OJFdroafW"
+      crossorigin="anonymous"
+    ></script>
+    <script type="module" src="../js/logout.js"></script>
+    <script type="module" src="../js/fetcher.js"></script>
```

Comment:
If you are using modules (imports/exports), which you are, you don't need to link to the files here. For example, you import and use fetcher in posts.js, you don't need to link to fetcher.js like this. The modules handle that.